packages/blob/src/test/index.ts

```diff
@@ -23,6 +25,8 @@ describe( 'getBlobTypeByURL', () => {
 	} );
 
 	it( 'returns undefined if the url is not defined', () => {
+		// calling getBlobTypeByURL without a URL is not type compliant, so ignore it
+		// @ts-ignore
 		expect( getBlobTypeByURL() ).toBe( undefined );
```

Comment:
We can also use a better undefined assertion here since we're touching this file:

```suggestion
		// calling getBlobTypeByURL without a URL is not type compliant, so ignore it
		// @ts-ignore
		expect( getBlobTypeByURL() ).toBeUndefined();
```